crates/oro-tree/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+use memmap::MmapOptions;
+use semver::{Version, VersionReq};
+use serde::{
+    de::{Deserializer, Error as SerdeError},
+    Deserialize,
+};
+use ssri::Integrity;
+use std::fs::File;
+use std::{collections::HashMap, path::Path};
+use thiserror::Error;
+
+fn default_as_false() -> bool {
+    false
+}
```

Comment:
You don't need this. `#[serde(default)]` will do the right thing without a specific default function provided.